src/chainparams.cpp

```diff
@@ -474,13 +474,6 @@ class CRegTestParams : public CChainParams {
         // Founders reward script expects a vector of 2-of-3 multisig addresses
         vFoundersRewardAddress = { "t2FwcEhFdNXuFMv1tcYwaBJtYVtMj8b1uTg" };
         assert(vFoundersRewardAddress.size() <= consensus.GetLastFoundersRewardBlockHeight());
-
-        // Enable Sprout shielded value pool checkpointing on
-        // regtest.
-        nSproutValuePoolCheckpointHeight = 0;
-        nSproutValuePoolCheckpointBalance = 0;
-        fZIP209Enabled = true;
```

Comment:
Nit: I'd prefer that we explicitly set `fZIP209Enabled = false;` here, as we do with other left-at-default flags.